sktime/forecasting/var.py

```diff
@@ -245,7 +245,9 @@ def _predict_interval(self, fh, X, coverage):
 
         for cov in coverage:
             alpha = 1 - cov
-
+            # A hacky way to coerce error-inducing alpha==1 into its approximant
+            if alpha >= 0.99999:
+                alpha = 0.99999
```

Comment:
Thoughts: can/should we document this hack for users, and if so, where?

Reply:
docstring?

Reply:
I think docstring modifications in these file do not get shown in read the docs. These are private methods (starting with _), and what all gets shown is only those of the base class.

Please let me know if this a misunderstanding.

Reply:
I mean, the class docstring, that gets overridden.